integrations/gradle-bloop/src/test/scala/bloop/integrations/gradle/ConfigGenerationSuite.scala

```diff
@@ -1586,7 +1585,8 @@ abstract class ConfigGenerationSuite {
     assert(Files.exists(configDir.toPath), "Does not exist: " + configDir)
     val configDirectory = AbsolutePath(configDir)
     val loadedProjects = BuildLoader.loadSynchronously(configDirectory, logger)
-    val build = Build(configDirectory, loadedProjects)
+    val workspaceSettings = WorkspaceSettings.readFromFile(configDirectory, logger)
+    val build = Build(configDirectory, loadedProjects, workspaceSettings)
     State.forTests(build, TestUtil.getCompilerCache(logger), logger)
```

Comment:
You might need to rebase/push merge commit @wiwa because the zinc submodule diff below should not show up